Goobi/src/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -1288,13 +1290,29 @@ public void setBatchHelper(BatchStepHelper batchHelper) {
     public List<ProcessProperty> getContainerlessProperties() {
         List<ProcessProperty> answer = new ArrayList<ProcessProperty>();
         for (ProcessProperty pp : this.processPropertyList) {
-            if (pp.getContainer() == 0) {
+            if (pp.getContainer() == 0 && (isEmptyString(pp.getValue())
+                    || !getOccupiedProperties().contains(pp.getName()))) {
                 answer.add(pp);
             }
         }
         return answer;
     }
 
+    private Set<String> getOccupiedProperties() {
+        Set<String> propertiesMitWerten = new HashSet<>();
+        for (ProcessProperty pp : processPropertyList)
+            if (isEmptyString(pp.getValue())) propertiesMitWerten.add(pp.getName());
+        return propertiesMitWerten;
+    }
+
+    /**
+     * @param theString the string
+     * @return true or false
+     */
+    private boolean isEmptyString(String theString) {
```

Comment:
You are testing for non empty value but you are naming the method for test of empty value. You should adjust the test or method name.

Reply:
Has done with it …